kinde_sdk/auth/oauth.py

```
@@ -1,633 +1,721 @@
-            "nonce": search_params["nonce"],
-            "code_challenge": search_params.get("code_challenge", ""),
-            "code_verifier": code_verifier,
+            'success': True,
+            'access_token': tokens_data.get('access_token'),
+            'refresh_token': tokens_data.get('refresh_token')
         }
+    except Exception as e:
+        return {'success': False, 'error': f'Refresh failed: {str(e)}'}
+"""
+from datetime import date
+from flask import Flask, url_for, render_template, request, session, jsonify, redirect
```

Ruff (0.11.9)

`flask.session` imported but unused

Remove unused import: `flask.session`

(F401)